src/js/components/addData/AddDataMeta.jsx

```diff
@@ -118,9 +126,35 @@ export default class AddDataMeta extends React.Component {
         }
     }
 
-    submitMetadata(){
-        this.props.updateMetaData(this.state);
-    }
+    submitMetadata(e){
+        let agency = this.state.agency, 
+            endDate = this.state.endDate,
+            dateType = this.state.dateType;
+
+        // Only make a request to check certified submission for quarterly submission. 
+        if (dateType == 'quarter') {
+        
+            let month = endDate.substr(0, 2), 
+                quarter = parseInt(month)%12 + 3,
+                year = endDate.substr(3);
+            
+            if (quarter == 3){
+                year++
+            }
+
+            AgencyHelper.checkYearQuarter(agency,year,quarter).then(()=>{
+                this.props.updateMetaData(this.state);
+            }).catch(err => {
+                let modalMessage = "An error occurred while attempting to create a submission. Please contact your administrator for assistance.";
```

Comment:
Why do you have this line here? this does nothing. You don't seem to use it anywhere.